### PR TITLE
Fix behavior indentation - close #1986

### LINKED ISSUE #1986: Cleanup malformed overindented behaviors
There are a number of behaviors throughout the codebase that are incorrectly indented e.g. [here in browser.cljs](https://github.com/LightTable/LightTable/blob/dec215a2846644e5544c971c4275f6d8313ad730/src/lt/objs/browser.cljs#L138-L173). Since there are still quite a few, we should just fix this and avoid the code churn it adds to pull requests. The PR for this should only be whitespace changes and will be confirmed with `git diff -w PR-branch master`

### PR DESCRIPTION
**Pull request by cldwalker**
This fixes the weirdly indented behaviors that have been scattered across LT's codebase. You can verify this is only whitespace changes by checking out the branch and `git diff -w master`. You shouldn't see any output as `-w` removes any whitespace changes.
@kenny-evitt @rundis Merging tomorrow night unless there are any concerns

### CODE REVIEW

File: src/lt/objs/app.cljs [hunk 1]
```diff
@@ -209,10 +209,10 @@
...
 
 (behavior ::add-platform-class
-                  :triggers #{:init}
-                  :reaction (fn [this]
-                              (dom/add-class (dom/$ :body) (name platform/platform))
-                              ))
+          :triggers #{:init}
+          :reaction (fn [this]
+                      (dom/add-class (dom/$ :body) (name platform/platform))
+                      ))
```
Comment (rundis) [inline src/lt/objs/app.cljs]: perhaps out of scope, but while you are at it :might be worth nuking a newline here -)

File: src/lt/objs/browser.cljs [hunk 2]
```diff
@@ -199,34 +199,34 @@
...
+                            {:label "copy"
+                             :order 3
+                             :click (fn [e]
+                                      (.copy (to-frame this)))}
+                            {:label "paste"
+                             :order 4
+                             :click (fn [e]
+                                      (.paste (to-frame this))
+                                      )}
+                            )
```
Comment (rundis) [inline src/lt/objs/browser.cljs]: newlines galore from here on :-)

File: src/lt/objs/browser.cljs [hunk 3]
```diff
@@ -257,104 +257,104 @@
...
 
 (behavior ::handle-send!
-                  :triggers #{:send!}
-                  :reaction (fn [this msg]
-                              (object/raise this (keyword (str (:command msg) "!")) msg)
-                              ))
+          :triggers #{:send!}
+          :reaction (fn [this msg]
+                      (object/raise this (keyword (str (:command msg) "!")) msg)
+                      ))
```
Comment (rundis) [inline src/lt/objs/browser.cljs]: newline

File: src/lt/objs/browser.cljs [hunk 4]
```diff
@@ -257,104 +257,104 @@
...
-                                (.send frame "editor.eval.css" #js {:name (:name info)
-                                                                    :code (:code info)})
-                                )))
+          :triggers #{:editor.eval.css!}
+          :reaction (fn [this msg]
+                      (let [info (:data msg)
+                            frame (to-frame (:frame @this))]
+                        (.send frame "editor.eval.css" #js {:name (:name info)
+                                                            :code (:code info)})
+                        )))
```
Comment (rundis) [inline src/lt/objs/browser.cljs]: - newline

File: src/lt/objs/clients/local.cljs [hunk 5]
```diff
@@ -56,10 +56,10 @@
...
 
 (behavior ::send!
-                  :triggers #{:send!}
-                  :reaction (fn [this data]
-                                (on-message (keyword (:command data)) (:data data) (:cb data))
-                              ))
+          :triggers #{:send!}
+          :reaction (fn [this data]
+                      (on-message (keyword (:command data)) (:data data) (:cb data))
+                      ))
```
Comment (rundis) [inline src/lt/objs/clients/local.cljs]: - newline

File: src/lt/objs/langs/behaviors.cljs [hunk 6]
```diff
@@ -94,33 +94,33 @@
...
+          :triggers #{:move}
+          :debounce 200
+          :reaction (fn [this]
+                      (let [idx (->index this)
+                            {:keys [tag behavior param-pos] :as info} (idx->entry-info idx (:entries @this))
+                            behavior-info (@object/behaviors behavior)]
+                        (if (:desc behavior-info)
+                          (object/raise helper :show! this behavior-info param-pos)
+                          (object/raise helper :clear!))
+                        )))
```
Comment (rundis) [inline src/lt/objs/langs/behaviors.cljs]: newline

File: src/lt/objs/opener.cljs [hunk 7]
```diff
@@ -135,70 +135,70 @@
...
-                                (cmd/exec! :save))
-                              ))
+          :triggers #{:blur}
+          :desc "Editor: Save on focus lost"
+          :type :user
+          :reaction (fn [this]
+                      (if (object/has-tag? this :editor)
+                        (object/raise this :save)
+                        (cmd/exec! :save))
+                      ))
```
Comment (rundis) [inline src/lt/objs/opener.cljs]: newline...

**Comment by rundis** [comment]
Looks good. Nice to see cleanup like this. Added some pointers to superfluous newlines, I mean since you were in whitespace nuking mode and all :-)

**Comment by cldwalker** [comment]
@rundis Agreed! Removing newlines and merging

**Pull request merged**